pkg/controller/clone-controller.go

```diff
@@ -768,14 +768,19 @@ func ValidateCanCloneSourceAndTargetContentType(sourcePvc, targetPvc *corev1.Per
 }
 
 // ValidateCanCloneSourceAndTargetSpec validates the specs passed in are compatible for cloning.
-func ValidateCanCloneSourceAndTargetSpec(sourceSpec, targetSpec *corev1.PersistentVolumeClaimSpec, contentType cdiv1.DataVolumeContentType) error {
-	err := ValidateCloneSize(sourceSpec.Resources, targetSpec.Resources)
-	if err != nil {
+func ValidateCanCloneSourceAndTargetSpec(sourcePvc, targetPvc *corev1.PersistentVolumeClaim, contentType cdiv1.DataVolumeContentType) error {
```

Comment:
why the change from ClaimSpec to Claim?

Reply:
So we can then check the `AnnPermissiveClone` annotation in the size validation-

Reply:
ok